transformers/struct.go

```diff
@@ -151,7 +153,10 @@ func TransformWithStruct(st any, opts ...StructTransformerOption) schema.Transfo
 				}
 			}
 		}
-
+		// Validate that all expected PK fields were found
+		if diff := funk.SubtractString(t.pkFields, t.pkFieldsFound); len(diff) > 0 {
```

Comment:
Is the order important here. can we just use set comparision?

Reply:
The diff is used to be able to provide an accurate message to the user

Reply:
Order doesn't seem important https://github.com/thoas/go-funk#funksubtract